packages/zowe-explorer-api/src/extend/MainframeInteraction.ts

```diff
@@ -119,13 +119,23 @@ export namespace MainframeInteraction {
          * @param {string} ussFilePath
          * @param {zosfiles.IDownloadOptions} options
          */
-        getContents(ussFilePath: string, options: zosfiles.IDownloadOptions): Promise<zosfiles.IZosFilesResponse>;
+        getContents(ussFilePath: string, options: zosfiles.IDownloadSingleOptions): Promise<zosfiles.IZosFilesResponse>;
```

Comment:
should we add a changelog item for this breaking change?

I believe this could impact some of the ongoing work happening in:
- #2783 

However, this PR should be merged first 😋